rsconf/component/mlflow.py

```diff
@@ -0,0 +1,68 @@
+"""MLFlow tracking server
+
+:copyright: Copyright (c) 2024 RadiaSoft LLC.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+
+from pykern import pkconfig
+from pykern.pkcollections import PKDict
+from pykern.pkdebug import pkdp
+from rsconf import component
+
+
+_DB_SUBDIR = "db"
+_MLFLOW_IP = "127.0.0.1"
```

Comment:
This should come from some place as LOCAL_HOST_IP. Maybe even pykern, because there's already something like this. The point isn't that it is changing, but rather that we know all the places we hardwire IPs to local host

Reply:
e52c947